internal/storage/v1/elasticsearch/spanstore/reader_test.go

```diff
@@ -311,40 +302,40 @@ func TestSpanReader_GetTrace(t *testing.T) {
 					{Hits: searchHits},
 				},
 			}, nil)
-		query := spanstore.GetTraceParameters{TraceID: model.NewTraceID(0, 1)}
+		query := []dbmodel.TraceID{dbmodel.TraceID(newTraceID(0, 1).String())}
```

Comment:
why does it matter which trace ID is used? Seems any string would do, e.g. `test-trace-id`